proteus/Domain.py

```diff
@@ -17,6 +19,17 @@ def __init__(self,nd,name="defaultDomain",units="m"):
         self.units=units
         self.x=[]#minx,miny,minz
         self.L=[]#bounding box when self.x is origin
+        self.g=g
+        # list of shape instances automatically attached to domain
+        self.shape_list = []
+        # list of auxiliaryVariables automatically attached to domaim
```

Comment:
I think we want the parameters below to  go somewhere else as well, if possible.